### PR TITLE
chore: Set up an Android CI job

### PR DESCRIPTION
**Pull request by robinlinden**
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/1992)

### CODE REVIEW

File: .github/scripts/cmake-android [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+cd ..
+
+# Build msgpack
+if [ ! -d msgpack-c ]; then
+  git clone --branch=c-4.0.0 https://github.com/msgpack/msgpack-c.git
+fi
+rm -rf msgpack-c/build
+mkdir -p msgpack-c/build
+cd msgpack-c/build
+cmake .. -DMSGPACK_BUILD_EXAMPLES=OFF -DCMAKE_TOOLCHAIN_FILE="$NDK"/build/cmake/android.toolchain.cmake -DANDROID_ABI="$ABI" -DCMAKE_INSTALL_PREFIX="$PREFIX" -DCMAKE_PREFIX_PATH="$PREFIX"
```
Comment (iphydf) [inline .github/scripts/cmake-android:61]: I'm guessing restyled did this, but can you make the quotes span the whole path, not just the `"$NDK"`?

Comment (iphydf) [inline .github/scripts/cmake-android:61]: Here and below: can you put one `-D` per line and indent like we do elsewhere?

**Review by iphydf — CHANGES_REQUESTED** [review]

File: .github/scripts/cmake-android (hunk 1 above)
Comment (robinlinden) [reply .github/scripts/cmake-android]: Done.

Comment (robinlinden) [reply .github/scripts/cmake-android]: Done.

**Pull request merged**